File: WebApplication/src/received-vouchers/dtos/received-voucher-communication.dto.ts
import { BaseCommunicationDto } from "src/signalR/notification/dtos/base-communication.dto";
import { ReceivedVoucherDto } from "./received-voucher.dto";

export class ReceivedVoucherCommunicationDto extends BaseCommunicationDto {

    entity: ReceivedVoucherDto;
}

File: Sc.Models/Dtos/ReceivedVouchers/ReceivedVoucherCommunicationDto.cs
using Sc.Models.Dtos.Notifications;

namespace Sc.Models.Dtos.ReceivedVouchers
{
    public class ReceivedVoucherCommunicationDto : BaseCommunicationDto
    {
        public ReceivedVoucherDto Entity { get; set; }
    }
}
